lms/djangoapps/lms_xblock/runtime.py

```diff
@@ -284,4 +284,8 @@ def applicable_aside_types(self, block):
         if block.scope_ids.block_type in config.disabled_blocks.split():
             return []
 
-        return super(LmsModuleSystem, self).applicable_aside_types()
+        return [
+            aside_type
+            for aside_type in super(LmsModuleSystem, self).applicable_aside_types(block)
+            if aside_type != 'acid_aside'
```

Comment:
And please reference https://openedx.atlassian.net/browse/TE-811 in that comment.